client/rpc_sender.go

```diff
@@ -83,6 +84,13 @@ func newRPCSender(server string, context *base.Context, retryOpts retry.Options,
 // and been executed successfully. We retry here to eventually get through with
 // the same client command ID and be given the cached response.
 func (s *rpcSender) Send(ctx context.Context, ba roachpb.BatchRequest) (*roachpb.BatchResponse, *roachpb.Error) {
+	timeout := 5 * s.retryOpts.MaxBackoff
```

Comment:
I'm not certain that that's where the cost of loading TLS certs is coming from. I think it is, but even if it's not, there must be some way to force the loading of the certs so we don't have to rely on a timeout that also covers all the unrelated portions of `Dial`).